units/systemd-fsck-root.service.in

```diff
@@ -11,8 +11,9 @@
 Description=File System Check on Root Device
 Documentation=man:systemd-fsck-root.service(8)
 DefaultDependencies=no
-Conflicts=shutdown.target
-Before=local-fs.target shutdown.target
```

Comment:
There's no need to change `Before=` - see 3acc3185913747be3bf86078d522029bea5898db